internal/api/database/file/file.go

```diff
@@ -141,7 +141,25 @@ func (d *DAO) RawQuery(query databaseModel.Query) ([]json.RawMessage, error) {
 		if readErr != nil {
 			return nil, readErr
 		}
-		result = append(result, data)
+
+		// If it's YAML, we need to convert to JSON first
+		if d.Extension == "yaml" {
```

Comment:
```suggestion
if d.Extension == config.YAMLExtension {
```

Reply:
@Nexucis  thank you for the suggestion. fixed in 6232c108bf11bb3e6fef9d658e7b8c1a6fa1cacb